docs/tdvt.md

```diff
@@ -461,27 +462,6 @@ The first thing TDVT does is try and connect to your database using the .tds fil
 
 1. Check tabquerytool.exe. This file should be placed in your Tableau bin directory and tdvt/config/tdvt_override.ini should be updated to point at that executable.
 
-__Tests fail with "Reference to undefined field..."__
-The first thing to check is that the column exists in your test database. If it doesn't, you will need to load the data again.
-
-TDVT expects the column names to exactly match those in the TestV1 datasets provided. It is recommended to match the column names exactly, as TDVT tests referencing those fields will fail otherwise.
-
-If for some reason the columns in your test database have to be named something different, you can work around that by adding column definitions to your TDS file that do match. For example, if your instance of the Staples table has the field Discounts with a trailing underscore, like so:
-
-```xml
-<column caption='Discount_' datatype='real' name='[discount_]' role='measure' type='quantitative' />
-```
-
-Add the following column definition:
-
-```xml
-<column datatype='real' name='[discount]' role='measure' type='quantitative'>
-    <calculation class='tableau' formula='[discount_]' />
-</column>
-```
-
-The new column will have the same data, and TDVT will be able to find it.
-
 __Boolean data types are not recognized or your database doesn't support them__
```

Comment:
This this section still valid advice then?

Reply:
It is, we have a couple of TDVT tests in teamcity that use the bool workaround and those pass, so the bool workaround does work and there are valid use cases.

I suspect that there's some part of logical query xml that breaks for most of the columns when they are calculated fields for some datatypes, but not for bools. It's likely possible to get the calculated field workaround working for all the types, but it may require changing the logical query xml for the logical tests, which is not a good idea.